app/presenters/outcome_presenter.rb

```diff
@@ -74,4 +73,19 @@ def render_data_partial(partial, variable_name)
     partial_path = ::SmartAnswer::FlowRegistry.instance.load_path.join("data_partials", "_#{partial}")
     ApplicationController.new.render_to_string(file: partial_path.to_s, layout: false, locals: {variable_name.to_sym => data})
   end
+
+  def render_erb_template
+    unless @rendered_erb_template
+      @view.render(template: erb_template_name, locals: @state.to_hash)
+      @rendered_erb_template = true
+    end
+  end
+
+  def has_content_for_body?
```

Comment:
I believe a common convention in ruby is to avoid `has_`, `is_` prefixes for methods that return a boolean value. A `?` is enough to indicate that.

As per http://xaviershay.github.io/writing/docs/ruby_style_guide.html

>Use ? suffix for methods rather than has_ or is_ prefix.↩
```
def valid?; "right" end
def is_valid?; "wrong" end
def has_valid?; "wrong" end
```